test-prow-e2e.sh

```diff
@@ -13,6 +13,20 @@ function copyArtifacts {
   fi
 }
 
+function waitForAuthOperatorProgressing {
+  output=
+  action="started"
+  if [ "$1" == "False" ]; then
+    action="finished"
+  fi
+  until [ "$output" == "$1" ]; do
+    output=$(oc get co/authentication -o 'jsonpath={.status.conditions[?(@.type=="Progressing")].status}' 2>&1)
+    echo "authentication operator hasn't $action Progressing 'test' idp; sleeping for a minute..."
+    sleep $((60 * 1))
```

Comment:
`60 * 1`? Why not just `sleep 60`? :)

I would probably poll a little more often, maybe every 15s.

Reply:
set to 15s